docs/source/resources/define-metadata.rst

```diff
@@ -1323,6 +1326,29 @@ contents.
          - *.some-extension
          - somefolder/*.some-extension
 
+Greater control over file matching may be
+achieved by defining ``files`` as a dictionary separating files to
+``include`` from those to ``exclude``.
+When using include/exclude, only files installed by
+the current recipe are considered. i.e. files in the prefix installed
+by host dependencies are excluded. include/exclude may not be used
```

Comment:
```suggestion
by host dependencies are excluded. include/exclude must cannot be used
```
`cannot` instead of `may` since the code is strict about the fact that both can't be used together?

Reply:
Just a small grammatical edit
```suggestion
by host dependencies are excluded. include/exclude must not be used
```